Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -267,7 +267,7 @@ public void addItemFlags(final String string) throws Exception {
     }
 
     public void addFireworkMeta(final CommandSource sender, final boolean allowShortName, final String string, final IEssentials ess) throws Exception {
-    if (MaterialUtil.isFirework(stack.getType())) {
+        if (MaterialUtil.isFirework(stack.getType())) {
```

Comment:
It looks like you changed the indents here and elsewhere in the file. Stick to the formatting in the original file.

Reply:
This still has different indentation than the original.